pmm-app/pr.codecept.js

```diff
@@ -2,8 +2,9 @@ exports.config = {
   output: 'tests/output',
   helpers: {
     Playwright: {
-      url: 'http://localhost/',
+      url: 'http://18.221.19.47/',
```

Comment:
Are we sure we want to commit this change?

Reply:
If yes, consider documenting what it is.

Reply:
thats my mistake, i'll fix that , thanks